actors/migration/nv16/test/miner_migration_test.go

```diff
@@ -146,11 +150,48 @@ func createMinersAndSectorsV7(t *testing.T, ctx context.Context, ctxStore adt.St
 	return append(minersToProve, minerInfos...), v
 }
 
+func makeManifest(t *testing.T, store cbor.IpldStore) cid.Cid {
+	adtStore := adt.WrapStore(context.Background(), store)
+	builder := cid.V1Builder{Codec: cid.Raw, MhType: mh.IDENTITY}
+
+	manifestData := manifest8.ManifestData{}
+	for _, name := range []string{"system", "init", "cron", "account", "storagepower", "storageminer", "storagemarket", "paymentchannel", "multisig", "reward", "verifiedregistry"} {
+		codeCid, err := builder.Sum([]byte(fmt.Sprintf("fil/8/%s", name)))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		manifestData.Entries = append(manifestData.Entries,
+			manifest8.ManifestEntry{
+				Name: name,
+				Code: codeCid,
+			})
+	}
+
+	manifestDataCid, err := adtStore.Put(context.Background(), &manifestData)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	manifest := manifest8.Manifest{
+		Version: 1,
+		Data:    manifestDataCid,
+	}
+
+	manifestCid, err := adtStore.Put(context.Background(), &manifest)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	return manifestCid
+}
+
 func TestNv16Migration(t *testing.T) {
```

Comment:
You should remove all the test logic specific to nv15 and replace it with a check that the system actor state looks ok.

Reply:
ok, will do.

Reply:
done.